File: Logic/3 - JavaScript/exercicio6.js
var sexo, altura, peso;
    sexo = prompt("Qual o sexo da pessoa estudada?\n1 - Feminino\n2 - Maculino");
    if((sexo==1) ||(sexo=="feminino")||(sexo=="FEMININO")||(sexo=="mulher")||(sexo=="MULHER")||(sexo=="Feminino")||(sexo=="Mulher"))
    {
        altura = prompt("Qual a altura dela(em centimetros)?");
        altura=altura/100
        peso=(72*altura)-58;
        alert("Se ela tem "+ altura + "m, o peso ideal dela é " + peso + "Kg.");
    }
    else if((sexo==2) ||(sexo=="masculino")||(sexo=="MASCULINO")||(sexo=="homem")||(sexo=="HOMEM")||(sexo=="Masculino")||(sexo=="Homem"))
    {
        altura = prompt("Qual a altura dele(em centimetros)?");
        altura=altura/100
        peso=(62*altura)-44;
        alert("Se ele tem "+ altura + "m, o peso ideal dele é " + peso + "Kg.");
    }
    else
    {
        alert("Desculpe, não consegui identificar o sexo da pessoa estudada, por favor, tente novamente")
    }

File: Logic/1 - C++/exercicio6.cpp
#include <stdio.h>
int main(void)
{
    float altura, peso;
    int sexo;
    printf("Qual o sexo da pessoa estudada?\n1 - Feminino\n2 - Maculino\n\n");
    scanf("%d", sexo);
    if(sexo==1)
    {
        printf("Qual a altura dela(em centimetros?");
        scanf("%f",&altura);
        altura=altura/100;
        peso=(72*altura)-58;
        printf("Se ela tem  %fm, o peso ideal dela é  %fKg.", altura, peso);
    }
    else if(sexo==2)
    {
        printf("Qual a altura dele(em centimetros?");
        scanf("%f",&altura);
        altura=altura/100;
        peso=(62*altura)-44;
        printf("Se ele tem  %fm, o peso ideal dele é  %fKg.", altura, peso);
    }
    else
    {
        printf("Desculpe, não consegui identificar o sexo da pessoa estudada, tente dígitar apenas números por favor");
    }

}